tests/eventEmitterListenerCount.test.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable max-len */
+
+'use strict';
+
+const { EventEmitter } = require('../src/MyEventEmitter');
+
+describe('EventEmitter "listenerCount" method tests', () => {
+  let emitter;
+
+  beforeEach(() => {
+    emitter = new EventEmitter();
+  });
+
+  test('should return the correct number of listeners for an event', () => {
+    const listener1 = jest.fn();
+    const listener2 = jest.fn();
+    const listener3 = jest.fn();
+
+    emitter.on('event', listener1);
+    emitter.on('event', listener2);
+    emitter.on('event', listener3);
+
+    const count = emitter.listenerCount('event');
+    expect(count).toBe(3);
```

Comment:
follow code style please
```suggestion
    const count = emitter.listenerCount('event');

    expect(count).toBe(3);
```